DebugServer/stdafx.h

```diff
@@ -19,3 +19,16 @@
 #include <Shlobj.h>
 
 #define BOOST_ALL_NO_LIB = 1
+
+// Ruby was configured for /Md builds. These macros are needed to
+// prevent compiler errors with Visual Studio when builds are compiled with MT.
+
+// Ruby 2.0
```

Comment:
This is an odd place for these defines. But I guess it works because we force-include stdafx.h from all cpp files.